sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/storage/block/FileCompactBlockRepository.kt

```diff
@@ -92,6 +90,24 @@ internal class FileCompactBlockRepository(
         return true
     }
 
+    override suspend fun deleteCompactBlockFiles(blocks: List<JniBlockMeta>): Boolean {
```

Comment:
Should documentation be added to indicate that true means all blocks do not exist, while false means that at least some blocks exist?  (I'm using "exist", because this method will return true if no deletion occurs as long as the files didn't exist in the first place)

Reply:
I just slightly updated the related interface documentation to reflect the file existence information as well.